tests/test_standard_tags.py

```diff
@@ -158,6 +164,9 @@ def test_route(self):
             tags["http.route"] = "/sample_rate_route/:i"
         if context.library == "dotnet":
             tags["http.route"] = "/sample_rate_route/{i:int}"
+        if context.library == "python":
+            if context.weblog_variant == "flask-poc":
+                tags["http.route"] = "/sample_rate_route/<i>"
```

Comment:
Well, this [failure](https://github.com/DataDog/system-tests/actions/runs/3418100849/jobs/5690011335#step:9:132) says that you need to add this : 

```python
            if context.weblog_variant in ("flask-poc", "uwsgi-poc"):
                tags["http.route"] = "/sample_rate_route/<i>"
            elif context.weblog_variant == "django-poc":
                tags["http.route"] = "sample_rate_route/<int:i>"
```